src/screens/Podcast/Podcast.tsx

```diff
@@ -1,112 +1,159 @@
-import React from 'react'
-import { View, Image, ImageBackground, TouchableOpacity } from 'react-native'
-import { ScrollView } from 'react-native-gesture-handler'
+import React, { useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import { useRoute } from '@react-navigation/native'
+import { useNavigation } from '@react-navigation/native'
+import { RouteProp } from '@react-navigation/native'
```

Comment:
Some problem with imports
```suggestion
import { useRoute, useNavigation, RouteProp } from '@react-navigation/native'
```